docs/filters.md

```diff
@@ -448,6 +448,74 @@ Examples:
 oauthTokeninfoAllKV("k1", "v1", "k2", "v2")
 ```
 
+## oauthTokenintrospectionAnyClaims
+
+If skipper is started with `-oauth2-issuer-url` flag, you can use this
+filter.
+
+The filter accepts variable arguments of strings, which are used to
```

Comment:
"...variable arguments of strings, ..." might be better as "...variable number of string arguments, ..."